addon/services/hot-reload.js

```diff
@@ -1,9 +1,62 @@
 import Ember from 'ember';
 
-export default Ember.Service.extend(Ember.Evented, {
-  init () {
-    //TODO: non global alternatives please :)
-    window.hotReloadService = this;
```

Comment:
Oh, I assumed the event handling (from JS to ember) required it + the dev tools plugin. That not the case ?